draft-ietf-ohai-ohttp.md

```diff
@@ -1002,6 +1010,11 @@ rate might choose to authenticate the proxy to enable the higher rate.
 
 ### Linkability Through Traffic Analysis {#ta}
 
+This document assumes that all communication between different entities is
+protected by HTTPS.  This protects information about which resources are the
```

Comment:
HTTPS is only required for client<>proxy communication to prevent this type of inference. Trying to clarify this might only complicate things, but this text seems to suggest that both hops need HTTPS. I don't know how I'd rewrite this any other way. 🤷